src/components/SyncServiceSignIn/stylesheet.css

```diff
@@ -5,17 +5,19 @@
   margin: 0;
 }
 
-#webdavLogin form {
+.sync-service-container form {
   display: table;
+  width: 100%;
 }
-#webdavLogin p {
+.sync-service-container form p {
   display: table-row;
 }
-#webdavLogin label {
+.sync-service-container form label {
   display: table-cell;
 }
-#webdavLogin input {
+.sync-service-container form input:not([type='submit']) {
   display: table-cell;
+  width: calc(100% - 10px);
```

Comment:
> I made a tiny CSS change here - the form inputs now go all the way to the right edge instead of leaving wasted space. GitLab project URLs tend to be pretty long. This applies to the WebDAV form as well for consistency.

It's a good change, for WebDAV, as well :+1: 

> Maybe one day we could rewrite this to use flexbox instead of 90s-style tables, but I didn't want to do that in this PR.

We haven't put a lot of effort into the forms. If you can/want to make them prettier, I'm all in favor! But did we really use tables? I can't find that code(;

Reply:
> But did we really use tables? I can't find that code(;

Yes, via CSS though. See https://developer.mozilla.org/en-US/docs/Web/CSS/display for `table`, `table-cell`, and `table-row`.

I may make a PR to improve it in the future if I get around to it. No promises and likely not soon though. If so though then it will just be straightforward modernization not drastic UI redesign so I don't think it will be controversial.

Reply:
:+1: